src/macaron/__main__.py

```diff
@@ -197,14 +197,31 @@ def verify_policy(verify_policy_args: argparse.Namespace) -> int:
         show_prelude(verify_policy_args.database)
         return os.EX_OK
 
+    policy_content = None
     if verify_policy_args.file:
         if not os.path.isfile(verify_policy_args.file):
             logger.critical('The policy file "%s" does not exist.', verify_policy_args.file)
             return os.EX_OSFILE
 
         with open(verify_policy_args.file, encoding="utf-8") as file:
             policy_content = file.read()
+    elif verify_policy_args.policy:
+        policy_dir = os.path.join(macaron.MACARON_PATH, "resources/policies/datalog")
```

Comment:
We avoid using `/` in strings and instead use the `os.path.join` method to construct the path.
```suggestion
        policy_dir = os.path.join(macaron.MACARON_PATH, "resources", "policies", "datalog")
```